ports/servo/glutin_app/window.rs

```diff
@@ -1321,6 +1323,21 @@ impl WindowMethods for Window {
                     self.event_queue.borrow_mut().push(WindowEvent::Reload(browser_id));
                 }
             }
+            (CMD_OR_CONTROL, Some('l'), _) => {
```

Comment:
@paulrouget this previous implementation (#17062) used (CMD_OR_CONTROL, _, Key::L) => { here in order to work on Windows. Did that ever get figured out because as far as I can tell this doesn't work in Windows 10?

Reply:
Right. https://github.com/servo/servo/issues/17146 is still open. I'm wondering if #19895 will fix this.